theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -87,6 +87,18 @@
                         {% endif %}
                     {% endif %}
                 {% endif %}
+
+                <a id="btn-download-all" 
```

Comment:
this `id="btn-download-all"` is a duplicate with the id from another template:
https://github.com/hydroshare/hydroshare/blob/2.15.9.1/hs_core/templates/pages/baseresource.html#L846

This will cause odd behavior when you click on the cloud in the top-right-buttons, because there is a jquery event listener registered to this id:

https://github.com/user-attachments/assets/a827c7ba-23fe-4f9c-b890-7faf402e1548

I recommend that, if we use a `download-spinner` onto this element in the top-right-buttons, it should overlay over the top of the button, not replace it. At any rate, I suggest you give this `<a>` a new id and modify the [event handler in hs_file_browser.js](https://github.com/hydroshare/hydroshare/blob/2.15.9.1/theme/static/js/hs_file_browser.js#L1855-L1858) to handle this new case

Reply:
Or, if @martinseul and others are okay with having the button _replaced_ by a spinner after clicking, then at the very least we should use a unique id for the new element you're adding (and add that id to the `click` handler so that it triggers as desired)